WFS/index.ts

```diff
@@ -0,0 +1,73 @@
+import { GeoJSON, FeatureCollection, BoundingBox, Feature, Geometry, Point } from '../GeoJSON';
+
+export class Parser {
+
+    propsToExtract = {
+        'nl': { zipCode: "BELB:adresse_zip", street: "BELB:adresse_txt", municipality: "BELB:gemeente", name: "BELB:nom", description: "BELB:descriptie", extra: { age: "BELB:leeftijd" } },
+        'fr': { zipCode: "BELB:adresse_zip", street: "BELB:adresse_txt", municipality: "BELB:commune", name: "BELB:nom", description: "BELB:description", extra: { age: "BELB:age" } }
+    };
+
+    constructor() {
+        let parser = new DOMParser();
+        fetch("https://cors-anywhere.herokuapp.com/https://wfs.environnement.brussels/belb?SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature&TYPENAME=playground&SRSNAME=EPSG:31370", { headers: { "Content-Type": "application/xml" } }).then(res => res.text()).then(text => {
```

Comment:
Using https://cors-anywhere.herokuapp.com/ otherwise request is blocked by browser.